### PR TITLE
Remaining mixer renames

### PR DESCRIPTION
**Pull request by jsarha**
Hopefully the final PR to complete "all controls and PCM names SHALL have a meaningful name."
in https://github.com/thesofproject/sof/issues/6576

This (and the previous PRs) all touch the PCM names, but does not change them. @plbossart if you would like to see some changes in the PCM names, you can comment that in this PR.

### CODE REVIEW

File: tools/topology/topology2/platform/intel/sdw-jack-generic.conf [hunk 1]
```diff
@@ -7,6 +7,11 @@ IncludeByKey.PASSTHROUGH {
 }
 }
 
+Define {
+       JACK_PLAYBACK_PCM_NAME	"Jack out"
+       JACK_CAPTURE_PCM_NAME	"Jack in"
```
Comment (plbossart) [inline tools/topology/topology2/platform/intel/sdw-jack-generic.conf:12]: Jack Out/In

**Review by plbossart — CHANGES_REQUESTED** [review]

File: tools/topology/topology2/platform/intel/sdw-jack-generic.conf (hunk 1 above)
Comment (jsarha) [reply tools/topology/topology2/platform/intel/sdw-jack-generic.conf]: Fixed in the latest version.

File: tools/topology/topology2/cavs-sdw-src-gain-mixin.conf [hunk 2]
```diff
@@ -33,6 +33,8 @@ Define {
 	SDW_CAPTURE_PCM		'SDW0-Capture'
 	PLAYBACK_LINK_ID	0
 	CAPTURE_LINK_ID		1
+	JACK_OUT_PCM_NAME	"Jack out"
+	JACK_IN_PCM_NAME	"Jack in"
```
[the remark this answers is not in the document]
Comment (jsarha) [reply tools/topology/topology2/cavs-sdw-src-gain-mixin.conf]: Fixed in the latest version.

**Comment by jsarha** [comment]
@plbossart and @ranj063 a gentle ping.

**Review by singalsu — APPROVED** [review]

**Comment by kv2019i** [comment]
Given this a topology2 change, I'll ignore the stuck Intel System CI tests, and I ignore the manifest-check as this does not change rimage (submodule or manifest version). I'll proceed with merge.

**Pull request merged**